src/install.ts

```diff
@@ -17,12 +17,13 @@ export async function runCommand(command: string): Promise<string> {
 }
 
 async function installOnLinux(version: string): Promise<string> {
-  const system = runCommand('uname -s')
-  const hardware = runCommand('uname -m')
+  let system = await runCommand('uname -s')
+  const hardware = await runCommand('uname -m')
```

Comment:
Seems rhis change is not so necessary. Let's keep the diff simpler.

Reply:
Okay. I thought the url generation below would be more unified this way.